packages/2018-transportation-systems/src/state/decline-in-ridership/actions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+import importAdapter from '../import-adapter';
+import actionEmitter from '../import-adapter-action-emitter';
+
+export const IMPORT_START = 'DECLINE_IN_RIDERSHIP_OVER_TIME/START';
+export const IMPORT_SUCCESS = 'DECLINE_IN_RIDERSHIP_OVER_TIME/SUCCESS';
```

Comment:
Since this is dealing with an API rather than a file, it's really important we handle the `ERROR` action. 

With the way this is written now, if the API request fails (i.e., responds with a 4xx or 5xx status code), the card will stay in the loading state and the user will have no indication that the request is no longer loading, it's just broken.